### PR TITLE
Content Trust more-info

### PR DESCRIPTION
**Pull request by pvizeli**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/more-info/unsupported/content_trust.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Ignored content trust"
+description: "More information on why disabling content-trust marks the installation as unsupported."
+---
+
+## The issue
+
+You have disable the content trust check. That means we can't veryify anymore if the content is trusted and was not modify by attackers.
+We strongly recomment to have this enabled.
```
Comment (klaasnicolaas) [inline source/more-info/unsupported/content_trust.markdown]: `recomment` or `recommend`? 🤔

File: source/more-info/unhealthy/untrusted.markdown [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+description: "More information on why untrusted content makes an installation as unhealthy."
+---
+
+## The issue
+
+We detect untrusted code. It look like someone/something try to inject not signed images or code. Don't worry, we block this action and nothing was apply.
+
+## The solution
+
+Please make sure your Network is not affected by an Hacker and try it again. For reset this status, you need restart the Supervisor.
```
Comment (spacegaier) [inline source/more-info/unhealthy/untrusted.markdown]:
```suggestion
---
title: "System detected untrusted content"
description: "More information on why untrusted content marks an installation as unhealthy."
---

## The issue

Home Assistant detected untrusted code. It looks like someone/something tried to inject not signed images or code into your Home Assistant system. But don't worry, we blocked this action and nothing was applied.

## The solution

Please make sure your network is not affected by an intruder and try it again. To reset this status, you need to restart the Supervisor.
```

File: source/more-info/unsupported/content_trust.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+```bash
+ha supervisor options --content-trust
+```
+
+When you have the content-trust, restart the Supervisor.
+
+```bash
+ha supervisor restart
+```
```
Comment (spacegaier) [inline source/more-info/unsupported/content_trust.markdown:25]:
```suggestion
---
title: "Ignored content trust"
description: "More information on why disabling content-trust marks the installation as unsupported."
---

## The issue

You have disabled the content-trust check. That means Home Assistant can't verify anymore if the content is trusted and was not modified by attackers.
We strongly recommend to have this check enabled.

For content-trust, we use the open source solution [CodeNotary](https://codenotary.io).

## The solution

To resolve this status, you need to re-enable the protection mechanism of content-trust by using the CLI:

```bash
ha supervisor options --content-trust
```

When you have enabled the content-trust, restart the Supervisor.

```bash
ha supervisor restart
```
```

Comment (pvizeli) [reply source/more-info/unsupported/content_trust.markdown:25]: look like the suggestion is corrupt?

Comment (spacegaier) [reply source/more-info/unsupported/content_trust.markdown:25]: I will delete and re-add

File: source/more-info/unsupported/content_trust.markdown [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+## The issue
+
+You have disable the content trust check. That means we can't veryify anymore if the content is trusted and was not modify by attackers.
+We strongly recomment to have this enabled.
+
+For Content-Trust, we use the OpenSource solution [CodeNotary](https://codenotary.io).
+
+## The solution
+
+To solve this you need to re-enable the protection mechanism by content-trust by using the CLI:
```
Comment (spacegaier) [inline source/more-info/unsupported/content_trust.markdown]:
```suggestion
## The issue

You have disabled the content-trust check. That means Home Assistant can't verify anymore if the content is trusted and was not modified by attackers.
We strongly recommend to have this check enabled.

For content-trust, we use the open source solution [CodeNotary](https://codenotary.io).

## The solution

To resolve this status, you need to re-enable the content-trust protection mechanism by using the CLI:
```

File: source/more-info/unsupported/content_trust.markdown [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+```bash
+ha supervisor options --content-trust
+```
+
+When you have the content-trust, restart the Supervisor.
```
Comment (spacegaier) [inline source/more-info/unsupported/content_trust.markdown]:
```suggestion
When you have enabled the content-trust, restart the Supervisor.
```

**Review by spacegaier — APPROVED** [review]

**Pull request merged**